sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/GeneratedFileFilterTest.java

```diff
@@ -58,13 +58,14 @@ public void setUp() {
       SonarRuntimeImpl.forSonarQube(Version.create(7, 9), SonarQubeSide.SERVER, SonarEdition.COMMUNITY)) {
     };
     MapSettings settings = new MapSettings(new PropertyDefinitions(definitions.create()));
-    defaultConfiguration = new AbstractSolutionConfiguration(settings.asConfig(), "cs") { };
+    defaultConfiguration = new AbstractSolutionConfiguration(settings.asConfig(), "cs") {

```

Comment:
is this change necessary?

Reply:
Functionally no. Auto-Formatting did this. I believe I've set it as it should be according to [sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style-configuration-for-intellij)

I'm not sure what settings does that, but [this line](https://github.com/SonarSource/sonar-developer-toolset/blob/7ef94ba29b0b47d1ccfd072bf926f75942b37fa1/eclipse/sonar-formatter.xml#L119) seems to be the one explicitly requiring this (since the `{ }` is an anonymous type declaration).